SPECS/plug-late-sr.spec

```
@@ -0,0 +1,34 @@
+Summary: Retry plugging PBDs to specific SRs on boot
+Name: plug-late-sr
+Version: 1.0
+Release: 1%{?dist}
+License: GPLv3
+Source0: plug-late-sr
+Source1: plug-late-sr.service
+BuildArch: noarch
+
+Requires: python2-configparser
+
+%description
+Retry plugging PBDs to specific SRs on boot.
```

How about adding a word about VMs?

```suggestion
Retry plugging PBDs to specific SRs on boot, then start VMs which couldn't autostart due to the unreachable SR.
```